index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="hamburger">
+        <div class='head'> 
```

Comment:
- [ ] OPTIONAL: As stated on the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), I suggest you be more consistent in your code when defining your classes.  Look:  

![image](https://user-images.githubusercontent.com/57421823/148509429-28e8ea08-3abc-42f7-a198-9bc1c06dcbe3.png)

I recommend you correct them when you have the opportunity. 😄